plugins/woocommerce/client/blocks/assets/js/blocks/classic-shortcode/editor.scss

```diff
@@ -49,6 +50,10 @@
 	.wp-block-woocommerce-classic-shortcode__placeholder-migration-button-container {
 		justify-content: center;
 		margin: $gap 0;
+		display: inline-flex;
+		flex-wrap: nowrap;
+		gap: $gap;
+		justify-content: flex-start;
```

Comment:
The later `justify-content` statement seems to overwrite the earlier one. I'd say, it's safe to get rid of the initial one.

Reply:
Good spot, done in https://github.com/woocommerce/woocommerce/pull/57095/commits/fa4372555bbb954ef556725126d5071262b0ec8d